form-functions.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+function gatherData(array $obj) :array {
```

Comment:
why have you named the param `$obj`? it is not an object, its an array as per your type hint

Reply:
also return types should be `functionName(): type` as per PSR2